ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+            max-width: $container-width;
+            margin: 0 auto;
+            color: #000000;
+
+            &.steps {
+                display: flex;
+                justify-content: space-between;
+            }
+        }
+
+        .step {
+            margin: 0 1.5rem 0 0;
+            font-family: Untitled-Sans-Regular;
+            font-size: 16px;
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            &__image {
+                width: 150px;
+                height: 150px;
+                display: block;
+                margin: 0 auto 26px;
+                position: relative;
+                left: -10%;
+            }
+
+            &__title {
+                display: inline;
+                font-family: Untitled-Sans-Medium;
+                letter-spacing: -0.07px;
+                line-height: 24px;
+                margin-right: 8px;
+            }
+
+            &__time {
+                letter-spacing: -0.07px;
+                line-height: 24px;
+            }
+
+            &__description {
+                font-size: 24px;
+                letter-spacing: -0.1px;
+                line-height: 28px;
+                margin-top: 16px;
+                max-width: 240px;
+            }
+        }
+    }
+
+    &.faq {
+        padding: 110px 0 60px 0;
+
+        .container {
+            box-sizing: border-box;
+            max-width: 900px;
+            padding: 0 60px;
+            margin: 56px auto;
+            text-align: left;
+        }
+    }
+
+    &.stay-informed {
+        padding: 86px 20px 72px 20px;
+
+        .container {
+            box-sizing: border-box;
+            max-width: $container-width;
+            margin: 0 auto;
+            text-align: left;
+        }
+
+        .social-media {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin-bottom: 40px;
+
+            .text {
+                text-align: left;
+                margin: 0 40px 0 0;
+                flex: 2 1 auto;
+                max-width: unset;
+            }
+
+            &__links {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                flex: 0 1 260px;
+            }
+
+            &__link {
+                flex: 0 1 50px;
+                text-align: center;
+            }
+        }
+
+        .title {
+            margin: 0;
+            padding-bottom: 18px;
+            text-align: left;
+        }
+    }
+
+    &.join-cmi {
+        padding: 76px 0 99px 0;
+
+        .join-cmi-logo {
+            max-width: 500px;
+        }
+
+        .title {
+            margin-top: 40px;
+            margin-bottom: 34px;
+        }
+
+        .text {
+            margin-bottom: 25px;
+        }
+
+        .join-cmi-btn {
+            margin: 0;
+        }
+    }
+}
+
+.title, .text {
+    max-width: 728px;
+    margin: 0 auto;
```

Comment:
consider keeping margin-bottom from browsers styles instead of adding for each title and text separately
margin-top: 0;
margin-right: auto;
margin-left: auto;

Reply:
However they (bottom margins) are different on mockups.